participants/Pufferfish Cloud Storage Drive.md

```diff
@@ -0,0 +1,53 @@
+We have three team work together and use different technique to access Filecoin storage pool. Now we'd like to use lotus and powergate attend SR2 competition. 
+
+## (2) Link to Project UI
+
+Who can use this web URL to access all features of Puffer Drive.
+http://www.pufferdrive.com
+
+## (3) What does your application/UI do?
+Puffer Drive front-end developed by Typescript + React + Ant Design 4.0 + OOP, User can use it's UI to upload/download/share/manage files. It's friendly.
+
+## (4) If your project is using a curated dataset, which one are you using?
+Our project currently didn't use any curated dataset, maybe after. We are using it for personal data cloud storage and public data storage and share.  
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+We have two data types, private and public. We don't have rights to access users' private data. We have exclusive rights to handle the shared public data. 
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We will invite our users to storage 100T+ public data during Slingshot competition.
+
+## (7) How are you structuring the data?
+All data is unstructured, we use hash to index them.
+
+## (8) What pre-processing are you doing before ingesting the data?
+We use database to storage and index all uploaded data. And provide public search feature. 
+
+## (9)  What tech stack will you use to store the data?
+We are planning to use lotus to access Filecoin storage physical infrastructure. Our another team will use Powergate.
+
+## (10) How will you retrieve the data?
+For private dataset, users' sign in their own account and browse directoried data who uploaded. As the public datase, users' will type search and our backend search engine find out related data hash to make retrieve access from Skynet Storage or Filecoin Storage. 
+
+## (11) Who is the intended user for your application/UI?
+We have hundreds thousands exists users already, we will invite and prize them attend this project.
+
+## (12) Do you have any users already or plans to acquire users soon?
+We have hundreds thousands exists users and will promote market acquire more new users attend our project.
+
+## (13) What challenges do you anticipate with this project?
+The concurrent retrieve bandwidth and speed is most factor to effect user experience. As far as we know, the current Filecoin storage pool has very poor internet bandwidth and the exist internet has interconnect problem between ISPs.

```

Comment:
We would recommend Powergate so that you can use both IPFS and Filecoin for storage for this application. Let us know if you have questions about this in the #powergate-users channel!

Reply:
Thank you so much to recommend Powergate. We studied it and decided to use it for our project. It is very useful and efficient way to interactive with Filecoin and IPFS.